sig-contributor-experience/annual-report-2021.md

```diff
@@ -4,79 +4,105 @@
+Contribex is a service and program orientated SIG. Most of our initiatives cover long term services for the Kubernetes project.
+
+
+  |         Subproject        |          Initiative / Program          |
+  |:-------------------------:|:--------------------------------------:|
+  | Community                 | Community Repo Stewardship             |
+  | Community Management      | Annual Contributor Survey              |
+  | Community Management      | Calendar Admin                         |
+  | Community Management      | Chair and TL Meetings + Docs           |
+  | Community Management      | Discuss Admin                          |
+  | Community Management      | Mailing List Admin                     |
+  | Community Management      | Slack Admin                            |
+  | Community Management      | Zoom / YouTube Admin                   |
+  | Contributor Documentation | Contributor Guide Stewardship          |
+  | Contributor Documentation | Contributor Site                       |
+  | Contributor Documentation | Developer Guide Audit                  |
+  | Contributor Documentation | Developer Guide Stewardship            |
+  | Contributors Comms        | Contributor / SIG Profiling            |
+  | Contributors Comms        | Stewardship of k8scontributors twitter |
+  | Devstats                  | Devstats Dashboard Update              |
+  | Events                    | Monthly Community Meeting              |
+  | Events                    | Office Hours                           |
+  | Events                    | Elections                              |
+  | GitHub Management         | GitHub Admin / Moderation              |
+  | GitHub Management         | GitHub Master -> Main rename           |
+  | GitHub Management         | GitHub New Membership Coordinator      |
+  | Mentoring                 | Google Summer of Code                  |
+  | Mentoring                 | Group Mentoring                        |
+  | Mentoring                 | LFX Mentor Program                     |
+  | Mentoring                 | Meet our Contributors                  |
+  | Mentoring                 | Outreachy                              |
+  | Slack Infra               | slack-infra                            |
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
-
-   -
-   -
-   -
+   
+   - GitHub Admin Subproject
+      - Looking to bring on more membership co-ordinators 
+   - Community Management Automation
+      - [Zoom to Youtube](https://github.com/kubernetes/community/issues/5201)
+      - [Workspace Automation](https://github.com/kubernetes/steering/issues/213)
+         - Mailing list management
+         - Calendar management
+         - Shared drives
+      - [Mailing list migration](https://github.com/kubernetes/community/issues/6353)
 
```

Comment:
Mentoring needs help, Having difficulty rescheduling the mentoring meeting,
Co-ordinating with outreachy, LFX & running mentoring cohorts.